specification/metadata/schema/schemaid.md

```diff
@@ -2,7 +2,7 @@
 
 The Schema ID provides the reference item to associate which Schema was used for the generation of a FOCUS Dataset.
 
-The SchemaId MUST be present in the metadata. The SchemaId MUST be of String. It is RECOMMENDED for SchemaId to be a Universally Unique Identifier (UUID) or [SemVer](https://semver.org) version.
+The SchemaId MUST be present in the metadata. The SchemaId MUST be of String. It is RECOMMENDED for SchemaId to be a Globally Unique Identifier (GUID).
```

Comment:
Should we emphasize that it MUST be unique within the context of a provider?

Reply:
Yeah we can strengthen this.

Reply:
this is done.